### PR TITLE
Update docstring for show_rag method

### PR DESCRIPTION
**Pull request by souravsingh**
Updates the docstring for ``show_rag`` method

Related to https://github.com/scikit-image/scikit-image/issues/2162

### CODE REVIEW

File: skimage/future/graph/rag.py [hunk 1]
```diff
@@ -422,10 +422,10 @@ def rag_boundary(labels, edge_map, connectivity=2):
 
 def show_rag(labels, rag, img, border_color='black', edge_width=1.5,
              edge_cmap='magma', img_cmap='bone', in_place=True, ax=None):
-    """Draw a Region Adjacency Graph on an image.
+    """Shows a Region Adjacency Graph on an image.
```
Comment (jni) [inline skimage/future/graph/rag.py]: Shows -> show. See [PEP 257](https://www.python.org/dev/peps/pep-0257/):

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

**Comment by souravsingh** [comment]
@jni Are there any other modifications needed?

File: skimage/future/graph/rag.py [hunk 2]
```diff
@@ -422,10 +422,10 @@ def rag_boundary(labels, edge_map, connectivity=2):
 
 def show_rag(labels, rag, img, border_color='black', edge_width=1.5,
              edge_cmap='magma', img_cmap='bone', in_place=True, ax=None):
-    """Draw a Region Adjacency Graph on an image.
+    """Show a Region Adjacency Graph on an image.
 
-    Given a labelled image and its corresponding RAG, draw the nodes and edges
-    of the RAG on the image with the specified colors. Edges are drawn between
+    Given a labelled image and its corresponding RAG,show the nodes and edges
```
Comment (soupault) [inline skimage/future/graph/rag.py]: Add a whitespace after the comma.

**Comment by souravsingh** [comment]
@jni Is the patch ready for merging?

**Comment by souravsingh** [comment]
Is the PR ready for merging?

**Comment by jni** [comment]
@souravsingh Sorry for the delay, I had a busy week last week. Here we go! =) Thanks for the contribution!

**Pull request merged**